src/components/ImageMessage.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { normalize } from '../config';
+import { HtmlView } from './HtmlView';
+
+export const ImageMessage = ({ message }) => (
+  <View style={styles.containerStyle}>
+    <HtmlView html={message} />
+  </View>
+);
+
+const styles = StyleSheet.create({
+  containerStyle: {
+    bottom: normalize(24),
+    paddingHorizontal: normalize(4),
```

Comment:
what does the paddingHorizontal do, that you cannot already do with increasing `bottom` to `normalize(28)` in this case?

Reply:
`horizontal` is "left-right", so a spacing from the container edges left and right. as you can see in the second example, this can be useful when having background color. sure, one can add that as well with html and nested divs, but we can provide is here already to not have to experiment too much in the html.

Reply:
uhm.. ye. left right xD
question still stands with `left: normalize(...)`

but this means that we have a little bit earlier line break for long texts right?

Reply:
if we increase the left, this does not affect the inner content spacing. so the background color and the text would start on the same pixel.

Reply:
but we have no background color, am i wrong?

Reply:
we can have like in example 2. and i think it will be a common use case, as just text on images are hard to read.
but yes, we should outsource it to the one who is adding the content.
as i  can see, i also added padding in the background div container haha `<div style=\"background-color: rgba(20, 20, 20, 0.4); padding: 4px\">` so the `paddingHorizontal` here could be removed, ok.

Reply:
done with https://github.com/ikuseiGmbH/smart-village-app-app/commit/e3b37da961f680deae9a4f7661148e6741f389f5